workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/WorkflowInterceptor.kt

```diff
@@ -98,6 +108,14 @@ public interface WorkflowInterceptor {
     session: WorkflowSession
   ): R = proceed(renderProps, renderState, null)
 
+  /**
+   * Intercept calls to [StatefulWorkflow.snapshotState] including the children calls.
+   */
+  public fun onSnapshotTree(
```

Comment:
This is a bit confusing. Why not `onSnapshotStateWithChildren`? Also could use kdoc explaining use case.

Reply:
done